### ISSUE TITLE
Q-Btn on Safari doesn't display properly when using :to

### ISSUE THREAD

**Issue by maggie44** [quasarframework/quasar#10636]
**Describe the bug**
When using `to` or `:to` on a q-btn the style changes when viewing through Safari (iPhone and Desktop Safari) (Firefox works ok), and doesn't seem to respond to the style requests.

<img width="119" alt="Screenshot 2021-09-10 at 08 36 39" src="https://user-images.githubusercontent.com/64841595/132880132-634a6b20-6aa0-4605-a7cc-a6b7381d760f.png">

**Codepen/jsFiddle/Codesandbox (required)**
Unfortunately I haven't been able to reproduce this in Codepen, it displays ok through their built in browser. But dropping the below code in a Quasar dev env and viewing through Safari produces the issue. 

**To Reproduce**

```
    <q-btn
      icon="folder_open"
      outline
      dense
      flat
      to="/"
    />
    <q-btn
      icon="folder_open"
      outline
      dense
      flat
    />
```

**Expected behavior**
Both these buttons should look the same like they do on Firefox (tested with Developer Edition):
<img width="208" alt="Screenshot 2021-09-10 at 08 54 59" src="https://user-images.githubusercontent.com/64841595/132882367-c2936500-462b-44ae-98ab-1a351d2748e7.png">

**Platform (please complete the following information):**
Quasar Version:
@quasar/app Version: 2.0.4

Safari:
Version 14.1.1 (16611.2.7.1.4)

**Comment by pdanpdan** [comment]
Please check the styles in Safari's inspector.
I cannot reproduce it.

**Comment by maggie44** [comment]
Poor investigation on my part, my apologies. 

Further investigation suggests it is Safari + Quasar + Tailwinds. The issue occurs as I mention above, but only when tailwinds is installed alongside Quasar. I'm not sure if this makes it a Quasar issue or a Tailwinds one, I will take your guidance on that. 

I have attached here a new project I created to reproduce the error. It is a Quasar project created using the Quasar CLI + a tailwinds setup as described here (https://dev.to/eltoritoelvis/add-tailwind-css-to-your-quasar-project-in-5-minutes-or-less-3n4o). Using Safari mobile and desktop the issue occurs, but not in Firefox. 

<img width="1063" alt="Screenshot 2021-09-18 at 17 03 43" src="https://user-images.githubusercontent.com/64841595/133911423-4ba94428-dbff-463a-b28e-e4138ed11be5.png">


[safari-issue.zip](https://github.com/quasarframework/quasar/files/7190792/safari-issue.zip)

**Comment by maggie44** [comment]
Is it viable to get this one marked as reopened?

**Comment by pdanpdan** [comment]
It's a style from tailwind that breaks it. Nothing we can do about it, sorry.

**Comment by maggie44** [comment]
> It's a style from tailwind that breaks it. Nothing we can do about it, sorry.

Thanks for checking it out. 

The Tailwind guys have identified an issue and suggested there is an issue with one of the attributes on the Quasar end. Here is the info: https://github.com/tailwindlabs/tailwindcss/discussions/5609#discussioncomment-1387116

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.1.3
